pkg/performanceprofile/controller/performanceprofile/components/components.go

```diff
@@ -27,10 +27,12 @@ type Options struct {
 }
```

Comment:
The commit message does not match the implementation, for example appending of the `shared` CPUs to the `reservedSystemCPUs` is not happening here and happened earlier. Am I missing something here?